### ISSUE TITLE
1.4.10 Reflow

### ISSUE THREAD

**Issue by MrSkippy** [developer-overheid-nl/don-site#188]
## 1.4.10 Reflow

**Gevonden probleem:**

Inzoomen op de pagina zorgt ervoor dat de elementen in de navigatie op geen ideale manier worden afgesneden. Zodra de tekst niet meer op 1 regel past, zou de navigatie moeten worden ingeklapt in het hamburger menu. Zie bijvoorbeeld: https://developer.overheid.nl/

Met zoom 400%, bij achteruit tabben vanaf de onderkant van de site worden bijvoorbeeld de GitHub, Slack en Discourse links niet getoond wanneer erop gefocust wordt. De viewport scrollt niet mee om deze gefocuste elementen te tonen. Zie bijvoorbeeld: https://developer.overheid.nl/kennisbank/apis/aan-de-slag/

Bij het naar boven tabben scrollt de inhoud niet hoog genoeg waardoor de toetsenbordgebruiker soms maar gedeeltelijk kan zien waarop ze gefocust zijn. Voorbeeld: De heading van de "Maak een OpenAPI" Card. Zie bijvoorbeeld: https://developer.overheid.nl/kennisbank/apis/aan-de-slag/

Het score blok op de onderste Card loopt buiten zijn container wanneer er wordt ingezoomd op de pagina. Zie bijvoorbeeld: https://apis.developer.overheid.nl/apis/cbs-cijfers

In Firefox, bij "Text only"-zoom, loopt de inhoud van het zoekpaneel buiten het paneel bij een zoom van 200%. Zie bijvoorbeeld: https://developer.overheid.nl/kennisbank/apis/

**Rapport:** 
https://toegankelijkheidsrapport.frameless.io/rapport/developer.overheid.nl-2025-04#sc-1.4.10

**Comment by MrSkippy** [comment]
Punt 1 opgelost in #213; Punt 2 en 3 opgelost in gekoppelde branch; Punt 4 meenemen in #199; Punt 5 wsch al opgelost in #48 / #157
